features/com.google.cloud.tools.eclipse.suite.feature/pom.xml

```diff
@@ -13,6 +13,13 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.eclipse.tycho</groupId>
+        <artifactId>tycho-packaging-plugin</artifactId>
+        <configuration>
```

Comment:
Is the feature version still based on the feature last-modification time? (I ask as I'm not sure the `<dependencies>` and `<timestampProvider>` are inherited?)